README.md

```diff
@@ -1,7 +1,9 @@
 # UiComponents [![Circle CI](https://circleci.com/gh/ad2games/ui_components.svg?style=svg)](https://circleci.com/gh/ad2games/ui_components)
 
 This gem provides common UI components for all
-[ad2games](http://www.ad2games.com/) projects.
+[ad2games](http://www.ad2games.com/) projects as well as related documentation.
+To view the documentation run the dummy app within the spec directory and
```

Comment:
Can we make this somehow easier? Always typing `cd spec/dummy; rails s` feels wrong.